website/docs/docs/building-a-dbt-project/metrics.md

```diff
@@ -29,6 +29,8 @@ A metric is a timeseries aggregation over a <Term id="table" /> that supports ze
 
 In v1.0, dbt supports metric definitions as a new node type. Like [exposures](exposures), metrics participate in the dbt DAG and can be expressed in YAML files. By defining metrics in dbt projects, you encode crucial business logic in tested, version-controlled code. Further, you can expose these metrics definitions to downstream tooling, which drives consistency and precision in metric reporting.
```

Comment:
[Non-blocking] Thoughts on updating this language? It still refers to:
- Metrics being brand-new in v1.0 — less true now than it was last December
- Metrics being **experimental** / liable to change in new minor versions — this is still true, but we anticipate removing this "experimental" designation in a few months!

Happy for those changes to come in a subsequent PR, since it's adjacent scope

Reply:
I agree - opened #1690 to remember this suggestion and track it in a future PR!